Python/Python.sublime-syntax

```diff
@@ -364,6 +364,12 @@ contexts:
 ###[ COMMENTS ]###############################################################
 
   comments:
+    - match: '(# ?)(TODO|ToDo|Todo|todo)\b(.*)'
```

Comment:
we generally try to avoid matching `.*` - instead we prefer to push to a different context, so syntax definitions which inherit or re-use these rules can do extra processing on that text (it allows `prototype`s to apply etc.)